src/components/AppContent/AppContent.vue

```diff
@@ -30,13 +30,40 @@
 </template>
 
 <script>
+import Hammer from 'hammerjs'
+import { emit } from '@nextcloud/event-bus'
 
 /**
  * App content container to be used for the main content of your app
  *
  */
 export default {
 	name: 'AppContent',
+
+	mounted() {
+		this.mc = new Hammer(this.$el, { cssProps: { userSelect: 'text' } })
+		this.mc.on('swipeleft swiperight', e => {
```

Comment:
Can we add a toggle for this, as for deck where we have horizontal scrolling we had quite some issues in the past when reordering cards with drag-and-drop caused the sidebar drag to trigger.

Reply:
Done!

Reply:
Awesome :rocket: